src/pages/Admin/Charity/Withdraws/Withdrawer/index.tsx

```diff
@@ -17,17 +23,30 @@ export default function Withdrawer() {
       }}
     >
       {({ tokens_on_hand }) => (
-        <AccountTabs
```

Comment:
@misicnenad, should not revert this abstraction, as this is used in strategies before which is now auto-invest once reimplemented. You could however move this to `/Withdrawer` as it is only used there atm

Reply:
@ap-justin it's unclear what you meant to say, do you mean that `AccountTabs` will be used again?

Reply:
currently removed as wiring is intended for `auto-invest` ,but would be brought back once we could query strategies from somewhere
<img width="649" alt="image" src="https://user-images.githubusercontent.com/89639563/229501466-a8e9ea35-0e21-4913-8dd8-e4e595b4cf29.png">

Reply:
> currently removed as wiring is intended for auto-invest

@ap-justin still unclear what you meant to say, **do you mean that AccountTabs will be used again?**

Reply:
@ap-justin anyway, this component seemed to be a wrapper around `headlessui > Tab`, but other than an additional level of abstraction it added nothing new - no custom styles nor some extra logic.

To use your way of thinking, let's use the `Tab` component directly for now and wrap it again when there is a need for that.

Reply:
@misicnenad  
 > nothing new other than an additional level of abstraction - no custom styles nor some extra logic.
 
 you mean for when I add back strategies tabs I need to add these boilerplate again? 
 ```ts
 const types: { [key in EndowmentType]: AccountType[] } = {
  charity: ["liquid", "locked"],
  normal: ["liquid"],
};
```
```ts
 {types[endow_type].map((type) => (
              <Tab.Panel key={`tab-panel-${type}`}>
```
or code back `<AccountTabs/>` component to bring these to pages the way there were before? 
```tsx
<AccounTabs>
  <Withdraw liquid />
  <Withdraw locked />
</AccountTabs>
```

```tsx
<AccounTabs>
  <Strategies liquid />
  <Strategies locked />
</AccountTabs>
```

Don't remove this component as there's a usecase for this which I just showed you. 

for this task you could even just simple return a single form 
```tsx
if(endow = normal)
 return <Withdraw type="liquid"/> 
 
 else return 
 <AccounTabs>
  <Withdraw liquid />
  <Withdraw locked />
</AccountTabs>

```

Reply:
@ap-justin The auto-invest feature was removed a while back from the contracts. We're not planning on adding that anymore since the cross-chain gas fees (both getting accurate estimates to pay & handling failed transactions) complicates things.